New version: ParameterEstimation v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: ParameterEstimation
- Repository: https://github.com/iliailmer/ParameterEstimation.jl
- Created by: @iliailmer
- Version: v0.3.0
- Commit: b778b42aeb1ea0c47798c690ba7e7a7eeafd5657
- Reviewed by: @iliailmer
- Reference: https://github.com/iliailmer/ParameterEstimation.jl/commit/b778b42aeb1ea0c47798c690ba7e7a7eeafd5657#commitcomment-134518403
- Description: ParameterEstimation.jl is a Julia package for estimating parameters and initial conditions of ODE models given measurement data.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6b94f0b544fe51c257bc1f3acff6ad167c5c8dc4ed711a8d2cf98ca43cf7f22a5fd33d6e7cad255761646b4ca19ce8577ab32f78d5895fa7ad18d61b8c53e97786c98bc9e7c716ccefb55d98b29293e8c69fe5c13d1bacca941a7c626d4644fcb7bb0b95ddad13000587bd64df592e0b4a4d455b14d231bf09e42d09eaee21a5a3ec10071e0cbc0ddc3808c0dda1829c1e51dd8e48a64d90355a05b62e36332036466b46ac7455ac929a0042d1e3ed9ad46fb3445e041f28d7c3f0c0ab7002334 -->

orebas commented:
(I have asked the same in the #pkg-registration channel on Slack):  Could this be manually merged with the current license?  The license has not changed and the package is already in the General Registry.  If not, please let me know how to proceed.

goerz commented:
It’s not true that the license hasn’t changed: https://github.com/iliailmer/ParameterEstimation.jl/commit/24c870e67c194340ef2b535043116f82b4d2bca9

I think the change itself is a violation of the original license (unless all contributors agreed), but more importantly: Cc-BY-NC is not suitable for software.

I do not think this should be merged

JuliaRegistrator committed:
New version: ParameterEstimation v0.3.0

UUID: b4cd1eb8-1e24-11e8-3319-93036a3eb9f3
Repo: https://github.com/iliailmer/ParameterEstimation.jl.git
Tree: 26098f996574c457f796ac5f888fd9f25d0d9dbe

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
